src/message_data_reader.py

```diff
@@ -1,22 +1,32 @@
-#This program can be used to print out everything in your Inbox or Sent folders and also take things out of the trash.
-#Scroll down to the bottom to see the functions that you can uncomment. Save then run this file.
-#The functions which only read the database file seem to function just fine even if you have Bitmessage running but you should definitly close it before running the functions that make changes (like taking items out of the trash).
+# pylint: disable=too-many-locals
+"""
+This program can be used to print out everything in your Inbox or Sent folders and also take things out of the trash.
+Scroll down to the bottom to see the functions that you can uncomment. Save then run this file.
+The functions which only read the database file seem to function just
+fine even if you have Bitmessage running but you should definitly close
+it before running the functions that make changes (like taking items out
+of the trash).
+"""
+
+from __future__ import absolute_import
 
 import sqlite3
```

Comment:
can we have imports first and from statements next.

Reply:
No. __future__ is special and should be at the top.